iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/receiver/protocol/pipeconsensus/PipeConsensusReceiver.java

```diff
@@ -1385,7 +1404,7 @@ private TPipeConsensusTransferResp onRequest(
             // when the last seal req is applied, we can discard this event.
             if (resp != null
                 && resp.getStatus().getCode() == TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
-              onSuccess(onSyncedCommitIndex + 1, isTransferTsFileSeal);
+              onSuccess(tCommitId, isTransferTsFileSeal);
```

Comment:
It looks like there's a +1 before there, is there any inconsistency?

Reply:
there is not inconsistency. tCommitId.getCommitIndex is always equal to onSyncedCommitIndex + 1 here